docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/deployment.md

```diff
@@ -241,8 +250,77 @@ Follow these instructions to update devices.
     [{"apiVersion":"v3","statusCode":201,"id":"fb5fb7f2-768b-4298-a916-d4779523c6b5"}]
     ```
     
-2. Map credentials using the `map-credentials.sh` script.  
+2. Map the credentials for the camera.
+
+!!! note
+    The preferred method for setting the credentials is through the use of the curl commands.
+
+=== "Curl Command"
+    !!! note
+        If running in secure mode all the api executions need the [JWT token](#token-generation) generated previously.
```

Comment:
Only the calls to the device service require the jwt (`/api/v3/secret`). other ones need the consul acl token.